### PR TITLE
Added guidelines before running 'npm run test' used to prevent test failure in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by MatteoGauthier**
Without babelify
image

To fix that :

```
npm run babelify
```

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ npm run style -- --fix
 
 ## Running tests
 
+The tests require the files to be compiled, you must first run babel transpilation :
```
Comment (geshii) [inline .github/CONTRIBUTING.md:27]:
```suggestion
The tests require the files to be compiled, you must first run the babel transpilation :
```
Just a small thing :v

**Review by vmfunc — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (MatteoGauthier) [reply .github/CONTRIBUTING.md]: Thanks :D

**Review by vmfunc — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -24,6 +24,12 @@ npm run style -- --fix
 
 ## Running tests
 
+The tests require the files to be compiled, you must first run the babel transpilation :
```
Comment (sachinraja) [inline .github/CONTRIBUTING.md:27]:
```suggestion
The tests require the files to be compiled, you must first run the babel transpilation:
```
May want to remove the space before the colon here.

Comment (MatteoGauthier) [reply .github/CONTRIBUTING.md:27]: Sorry I'm french 😉

**Review by vmfunc — APPROVED** [review]

**Comment by adamwathan** [comment]
Thanks!

**Pull request merged**